content/en/docs/rules/style-guide.md

```diff
@@ -167,15 +175,15 @@ When writing a rule for container workloads, you should include the fields we au
 %container.info
 ```
 
-This special placeholder will be automatically replaced with common container-related fields and, when used in conjunction with `-pk` (see the [output formatting](/docs/outputs/formatting/) page for more details) it will also add [basic Kubernetes fields](/docs/reference/rules/supported-fields/#field-class-k8s) (this does *not* require the *Kubernetes Metadata Enrichment*, i.e. `-k/-K`, functionality to be enabled). Using `%container.info` in conjunction with `-pk` is equivalent to:
+This special placeholder will be automatically replaced with common container-related fields and, when used in conjunction with `-pk` (see the [Output Formatting](/docs/outputs/formatting/) page for more details) it will also add [Basic Kubernetes Fields](/docs/reference/rules/supported-fields/#field-class-k8s). Using `%container.info` in conjunction with `-pk` is equivalent to:
```

Comment:
Does it work with both -pk and -pc? Or is that not relevant to this section?

Reply:
@leogr more ideas here?

Reply:
Yes, it works both for `-pk` and `-pc`. Not sure if it is really relevant here.

To summarize:

`-pc` is `container_id=%container.id container_image=%container.image.repository container_image_tag=%container.image.tag container_name=%container.name`

`-pk` is `-pc` plus `k8s_ns=%k8s.ns.name k8s_pod_name=%k8s.pod.name`

(implementation is here: https://github.com/falcosecurity/falco/blob/master/userspace/falco/app/actions/init_falco_engine.cpp#L24)

In both cases:
> If a rule's output contains %container.info, it will be replaced with the corresponding details. Otherwise, these details will be directly appended to the rule's output.

This is documented in the [Output Formatting](https://falco.org/docs/outputs/formatting/) page (and in the Falco inline command line help), so I'm not sure if it's worth duplicateing the content here.

Reply:
Tried to simplify it let me know